### PR TITLE
Accept Shipment Button

### PR DESCRIPTION
**Pull request by chrisgilmerproj**
## Description

This modifies the Accept/Reject buttons to be a component.  When the shipment Accept button is clicked it will send a POST request out to try and accept this shipment.  Accept/Reject buttons will hide once they are clicked and or the shipment state changes from 'AWARDED' to something else.

## Reviewer Notes

Please look at the error handling from the API call.

## Setup

```sh
./bin/generate-test-data -scenario=7
```

## Code Review Verification Steps

* [x] There are no aXe warnings for UI.
* [x] This works in IE.
* [x] Request review from a member of a different team.
* [x] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/n/projects/2181745/stories/159901428) for this change

## Screenshots

Here's what you see when a shipment is in the 'AWARDED' state:

screen shot 2018-08-23 at 1 59 02 pm

If you accept a shipment you see:
screen shot 2018-08-28 at 11 33 09 am

### CODE REVIEW

**Comment by chrisgilmerproj** [comment]
Yeah @kilbergr! Let's chat. I'm not sure this will work as promised until https://github.com/transcom/mymove/pull/911 is merged.  But it may be in a good enough place to just land since its mostly wired up.  If you're around let's try to do a zoom on this before we move forward.

**Comment by chrisgilmerproj** [comment]
After talking with @kilbergr there are still a couple things left to do on this.  She's going to take it over and finish it up.

File: src/scenes/TransportationServiceProvider/ShipmentInfo.jsx [hunk 1]
```diff
@@ -53,24 +91,29 @@ class ShipmentInfo extends Component {
...
+                  <PremoveSurvey
+                    title="Premove Survey"
+                    shipment={this.props.shipment}
+                    update={this.props.patchShipment}
+                  />
+                </div>
+              )}
+            </div>
+            <div className="usa-width-one-third">
+              {this.props.shipment.status === 'AWARDED' && (
```
Comment (macrael) [inline src/scenes/TransportationServiceProvider/ShipmentInfo.jsx]: I don't think we should be checking for AWARDED twice, either we shouldn't load the AcceptShipmentPanel at all (here) if it's not AWARDED, or the AcceptShipmentPanel should be responsible for what it displays itself, but not both.

Comment (chrisgilmerproj) [reply src/scenes/TransportationServiceProvider/ShipmentInfo.jsx]: 👍 - I think that was my change that @kilbergr inherited.  Good catch.

**Pull request merged**